touchnet/data/functions.py

```
@@ -0,0 +1,316 @@
+    """
+    for sample in data:
+        if 'txt' in sample:
+            # NOTE(xcsong): add bos/eos in batch_xxx()
+            input_ids = tokenizer.tokenize(sample['txt'], add_special_tokens=False)
+            sample['input_ids'] = input_ids
+            yield sample
+        else:
+            yield sample  # do nothing
+
+
+def filter(data, config: DataConfig):
```

Pylint (3.3.7)

[warning] Redefining built-in 'filter'

(W0622)